### PR TITLE
[14.0][IMP] rma: Auto-calculate package for serial products

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**
When returning or delivering a serial product from a RMA, we will calculate by default its package. Later, it can be modified by the user.

### CODE REVIEW

**Review by MateuGForgeFlow — APPROVED** [review]

File: rma/wizards/rma_make_picking.py [hunk 1]
```diff
@@ -206,49 +206,51 @@ def action_create_picking(self):
         else:
             pickings = self.mapped("item_ids.line_id")._get_in_pickings()
             action = self.item_ids.line_id.action_view_in_shipments()
-        if picking_type == "incoming":
```
Comment (AaronHForgeFlow) [inline rma/wizards/rma_make_picking.py:209]: why is this the 'if' removed?

Comment (BernatPForgeFlow) [reply rma/wizards/rma_make_picking.py:209]: With this if, you would never reach this point: https://github.com/ForgeFlow/stock-rma/blob/14.0/rma/wizards/rma_make_picking.py#L238

So, the 'for' loop should be for the incoming and outgoing. Then, for the serial products, we need to calculate lot_id, quantities and package regardless of whether it is incoming or outgoing. If not, inconsistencies are created in the stock quantities.

**Review by AaronHForgeFlow — APPROVED** [review]
ok thanks LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests also

**Comment by AaronHForgeFlow** [comment]
@BernatPForgeFlow  could forward port this?

**Pull request merged**